org.sparkleshare.SparkleShare.json

```diff
@@ -0,0 +1,136 @@
+{
+    "app-id": "org.sparkleshare.Sparkleshare",
+    "branch": "master",
```

Comment:
Remove this line. Our CI automatically sets branch to `stable`.

Reply:
I'll set it to "stable" but will keep it as it makes sense for builds outside flathub as well.